source/data_mining/embedTweets.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Reads files of [[tweet,date]] Applies Universal Sentence Encoder to to the
+tweet to produce a third column with a 512 component embedding.  Writes a json
+output file containing this three column data.
+
+June, 2019
+@author: Joshua Rubin
+"""
+
+import os
+import json
+import pandas as pd
+import tensorflow as tf
+import tensorflow_hub as hub
+
+# Retrieve Universal Eentence Sncoder.
+module_url = "https://tfhub.dev/google/universal-sentence-encoder/2"
+embed = hub.Module(module_url)
+
+def embedTweets(inputDirectory, outputDirectory): 
+    
+    # Grab the files from the input directory
+    tweetFiles = [f for f in os.listdir(inputDirectory) if f[0]=='@']
+        
+    for fileName in tweetFiles:   
+            
+        print('Loading ' + fileName + '.', end=' ')
```

Comment:
Check out f strings in python 3. `print(f"Loading {filename}....`

Reply:
Yesh.  I will be using that.